### PR TITLE
Add docs to repo.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
polar bear

### CODE REVIEW

File: sass/partials/_tools.scss [hunk 1]
```diff
@@ -12,6 +12,8 @@
 ///   A CSS `calc()` function that will toggle the proper values
 ///   based on the value of `--ccs-mode--zero` and `--ccs-invert--zero`
 /// @example scss - assign generates theme values that toggle
+///   @use './' as ccs;
```
Comment (jgerigmeyer) [inline sass/partials/_tools.scss]: @mirisuzanne I realize this isn't ideal. If we get a new Herman release out that supports `as`, then we can get rid of this.

File: package.json [hunk 2]
```diff
@@ -2,23 +2,27 @@
   "name": "cascading-color-systems",
   "version": "0.1.0-beta.11",
   "description": "generate dynamic color palettes with custom properties",
+  "title": "Cascading Color Systems",
   "main": "dist.js",
   "module": "index.js",
-  "homepage": "https://cascading-colors.netlify.com",
-  "repository": "mirisuzanne/cascading-color-system",
+  "homepage": "https://oddbird.net/cascading-colors/",
```
Comment (jgerigmeyer) [inline package.json]: @mirisuzanne Hm, maybe we should actually hold off on this particular change until OddLeventy is launched?

Comment (mirisuzanne) [reply package.json]: yeah, probably.

File: sass/partials/_tools.scss (hunk 1 above)
Comment (mirisuzanne) [reply sass/partials/_tools.scss]: I'll take a look - I bet there's a decent workaround

**Review by mirisuzanne — APPROVED** [review]

File: sass/tools/_modes.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+///       $dark: var(--ccs-prime--config)
+///     )};
+///     --ccs-custom-hue: none;
+///   }
+/// @group tools-modes
+@function if-mode($light, $dark) {
+  @return calc(
+    var(--ccs-mode--zero) * #{$light} + var(--ccs-invert--zero) * #{$dark}
+  );
+}
```
Comment (jgerigmeyer) [inline sass/tools/_modes.scss]: @mirisuzanne Is this file a duplicate of `partials/_tools.scss`?

Comment (mirisuzanne) [reply sass/tools/_modes.scss]: oh, it's because you made changes to one, after I had moved it. Fixing…

File: index.html [hunk 4]
```diff
@@ -118,7 +118,7 @@ <h2 data-ccs-colors="invert">neutral</h2>
   </main>
 
   <footer>
-    <p>Copyright © 2019 <a href="https://miriam.codes/">Miriam Suzanne</a></p>
+    <p>Copyright © 2019–2020 <a href="https://oddbird.net/">OddBird</a></p>
```
Comment (jgerigmeyer) [inline index.html:121]: @mirisuzanne I don't actually care who holds the copyright, but think this should match the LICENSE.md file.

Comment (mirisuzanne) [reply index.html:121]: I agree - and OddBird is better for it

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## UNRELEASED
+
+- INTERNAL: Upgrade dev dependencies
+- DOCS: Added documentation to repo
```
Comment (jgerigmeyer) [inline CHANGELOG.md:8]: @mirisuzanne Any additional changelog items we should add for this PR?

Comment (mirisuzanne) [reply CHANGELOG.md:8]: We re-organized a few partials internally, and some of the docs – but no changes to documented features. Adding one note from that…

**Review by jgerigmeyer — COMMENTED** [review]
👍

File: CHANGELOG.md (hunk 5 above)
Comment (mirisuzanne) [reply CHANGELOG.md:8]: K, not ready to document that as a public feature yet - but good to have it in the changelog. Should be good to go.

**Pull request merged**